packages/bot/src/views/commit.pug

```diff
@@ -0,0 +1,42 @@
+extends layout.pug
+block main
+  .container.mt-2
+    .row
+      .col-12
+        h3 #{commit.repo_name}@#{commit.id.slice(0, 8)}
+        p.lead "#{commit.message}"
+        p 
+          strong Committed at: 
```

Comment:
Using `strong` is not appropriate here, since we do not want to emphasize it, but indicate it as header label of an information field. We should either use `label` tag, or the rarer `dl` tag.